packages/mdc-slider/mdc-slider.scss

```diff
@@ -109,6 +155,37 @@ $mdc-slider-dark-theme-grey: #5c5c5c;
     border-radius: 50%;
     opacity: 0;
   }
+
+  &__pin {
+    @include mdc-theme-prop(background-color, primary);
+    @include mdc-theme-prop(color, text-primary-on-primary);
+
+    display: flex;
+    position: absolute;
+    top: 0;
+    left: 0;
+    align-items: center;
+    justify-content: center;
+    width: 26px;
+    height: 26px;
+    margin-top: -2px;
+    margin-left: -2px;
+    transform: rotate(-45deg) scale(0) translate(0, 0);
+    transition: transform 100ms ease-out;
```

Comment:
Good question, I don't think it is specified on spec. I will ask our motion design for some suggestion!